src/reflect/scala/reflect/internal/Flags.scala

```diff
@@ -169,6 +169,11 @@ class Flags extends ModifierFlags {
 
   final val SYNCHRONIZED  = 1L << 45      // symbol is a method which should be marked ACC_SYNCHRONIZED
 
+  final val SYNTHESIZE_IMPL_IN_SUBCLASS = 1L << 50 // used in fields phase to indicate this accessor should receive an implementation in a subclass
+
+  // flags used strictly internally in the Fields phase (info/tree transform):
+  final val NEEDS_TREES   = 1L << 59           // this symbol needs a tree. (distinct from SYNTHESIZE_IMPL_IN_SUBCLASS)
+
```

Comment:
these new flags seem to be only used within the fields phase, and they are not persisted since we're already past pickler. since flags are a bit brittle, could we instead use a symbol attachment?

if we keep them, we should update the comment just below here accordingly, and maybe update the `InitialFlags` mask.